docs/wiki/research/PBS/ePBS.md

```diff
@@ -0,0 +1,398 @@
+# Enshrined Proposer-Builder Separation (ePBS)
+
+## [DRAFT MODE - This is work in progress.]
```

Comment:
Is it still WIP?

Reply:
I want to write a brief section for PEPC similar to PTC and TBHL and remove WIP label. 

Is that ok?

Reply:
Of course, I was just checking